js/content/community.js

```diff
@@ -291,6 +292,260 @@ let CommentHandler = (function(){
         observer.observe(document.body, { childList: true });
     };
 
+    function loadScripts() {
+        return new Promise(async function(res) {
+            scriptsLoaded = true;
+            await ExtensionLayer.loadInPageContext("//steamcommunity-a.akamaihd.net/public/shared/javascript/shared_global.js?v=NuCEF6NW8c0Q");
+            await ExtensionLayer.loadInPageContext("//steamcommunity-a.akamaihd.net/public/javascript/livepipe.js?v=.sk9HEaDHE9C5");
+            await ExtensionLayer.loadInPageContext("//steamcommunity-a.akamaihd.net/public/javascript/textarea.js?v=.KmmHJqTpwrPO");
+            await ExtensionLayer.loadInPageContext("//steamcommunity-a.akamaihd.net/public/javascript/sharedfiles_editor.js?v=pqvj6_7nvfqb");
+            res();
+        })
+    }
+
+    async function addEditors() {
+        if (!scriptsLoaded) { await loadScripts(); }
```

Comment:
This is a race condition. loadScripts() could potentially be called twice.

Reply:
Hence, I use `scriptsLoaded` to avoid that.

Reply:
You're calling loadScripts(), and loadScripts() doesn't set scriptsLoaded until after it's created a Promise that creates a Promise. If any of these tasklets get deferred to the end of the current event stack, there's a 4-50ms window where this could *potentially* call loadScripts() again.